src/toast/ops/conviqt.py

```diff
@@ -750,30 +755,42 @@ def _exec(self, data, detectors=None, **kwargs):
         for det in all_detectors:
             verbose = self.comm.rank == 0 and self.verbosity > 0
 
+            # Find one process that has focalplane data for this detector
+            # and broadcast the focalplane properties
```

Comment:
In general, the same detector could have different properties from one observation to the next.  For example, a different pointing solution.  This technique is OK as long as the properties used later to select the beam file name are constant across the data set.